features/user_can_login_using_spotify.feature

```diff
@@ -1,17 +1,17 @@
 Feature: user can log in using Spotify credentials
-As a user of the Application
-In order to simplify the sign in/ sign up process
-I would like to be able to authenticate using my Spotify account
+  As a user of the Application
```

Comment:
Avoid modifying files that aren't related to the feature you're currently working on. This adds unnecessary clutter to the diff of this PR and makes it a bit hard to focus on what really matters here. Additionally, this can be cause for merge conflicts. So next time don't change them.